mobile-app/app/screens/AppNavigator/screens/Loans/components/LoanCardsV2.tsx

```diff
@@ -86,57 +110,109 @@ export function LoanCardsV2(props: LoanCardsProps): JSX.Element {
   ) as LoanVaultActive;
 
   return (
-    /* Known intermittent issue wherein the two-column layout is not followed in web - FlashList */
-    <ThemedFlashList
-      estimatedItemSize={116}
-      contentContainerStyle={tailwind("pt-4 pb-2")}
-      data={filteredLoanTokens}
-      /* This tells FlashList to rerender if any of the props below is updated */
-      extraData={{
-        isVaultReady,
-        activeVault,
-      }}
+    <ThemedScrollViewV2
+      contentContainerStyle={tailwind("px-5 py-8 w-full")}
```

Comment:
we cannot put the padding px-5 directly on the parent since the LoanCard have additional padding

```suggestion
      contentContainerStyle={tailwind("py-8 w-full")}
```